app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -16,6 +21,9 @@ import Tabs from "~/components/ui/controls/Tabs";
 import DetailsSidebar from "~/components/common/DetailsSidebar";
 import Controls from "./Controls";
 import PipelineVersionSelect from "./PipelineVersionSelect";
+import { SaveButton, ShareButton } from "~ui/controls/buttons";
+import { Popup } from "semantic-ui-react";
```

Comment:
Use our Popup wrapper in ui. 
As a common practice, we wrap all elements with our own. This allows us to add custom styling and some common functionality to the elements.
The Popup from semantic ui has other issues as, because of the way it is used, always renders with the whole page (the impact depends on where you use it).

Reply:
Ok good to know!